src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -197,12 +197,55 @@ void GraphingCalculator::OnEquationsVectorChanged(IObservableVector<EquationView
     GraphingControl->PlotGraph(false);
 }
 
+wstringstream GraphingCalculator::FormatTraceValue(double min, double max, float pointValue)
+{
+    wstringstream traceValueString;
+
+    // This formula will give us the precision to use for the pointValue based on the min/max. For example: 
+    // If we want to round to two decimal places, this will return 0.01
+    // If we want to round to the nearest 100th this will return 100
+    auto roundingNumber = pow(10, floor(log10(max - min)) - 3);
+
+    // Determine if we want to show scientific notation instead
+    if (roundingNumber <= 0.0000001 || roundingNumber >= 10000000)
+    { 
+        traceValueString << scientific;
+    }
+    else
+    {
+        traceValueString << fixed;
+    }
+
+    // If we are rounding to a decimal place, set the precision
+    if (roundingNumber < 1)
+    {
+        auto roundingNumberStr = std::to_wstring(roundingNumber);
+        TrimTrailingZeros(roundingNumberStr);
+        auto precisionLength = roundingNumberStr.length() - 2;
+        traceValueString << setprecision(precisionLength) << pointValue;
+    }
+    else
+    {
+        // If we are rounding to a place larger than 1, do that rounding here
+        if (roundingNumber > 1)
```

Comment:
Why display an inaccurate number, knowing that we will use the same number of characters?

Reply:
The reason is because it is hard to increment on smaller digits in the larger scales with your mouse. So to reduce noise while scanning with the mouse we round. I could be convinced to just display the full value (without decimals) if you feel strongly. We should consult with @grochocki too.

Reply:
It's commonly accepted to simplify the mantissa of a number, but not the characteristic part (digits before decimal separator), even if the reasoning is the same,

Reply:
I don't feel too strongly either way. @grochocki thoughts?

Reply:
If I understand Rudy's feedback properly here, he is suggesting we never round digits before the decimal separator. The intended behavior here is that when it is no longer possible to make precise tracing movements at the appropriate power of ten, we should bump the rounding to the next power of ten. As the graph output window scales, we will become increasingly sporadic in displayed tracing values due to imprecise nature of pixel to coordinate space mapping. While it would be computationally accurate, is not pragmatic for users, because they are not able to move the mouse and see massive jumps in output. To see more precise x/y values, users can simply zoom in. Or perhaps a future improvement might be to surface a way for users to input an exact x or y value to see the other.

Reply:
@grochocki: I'm 100% OK with rounding X, but we should recalculate Y to be consistent with the new value of X and not round X **and** Y and display incorrect pair of values

Reply:
X = 11111111111
Y = X*2 + 3

current behavior:
11111111111, 22222222225
this PR behavior:
11111110000, 22222220000 -> which is incorrect
Expected behavior:
11111110000, 22222220003